tutorial/conan_repositories/uploading_packages.rst

```diff
@@ -0,0 +1,81 @@
+.. _uploading_packages:
+
+Uploading Packages
+==================
+
+In the previous section we learned how to :ref:`set up a Conan repository
+<setting_up_conan_remotes>`. Now we will go through the process of uploading both recipes
+and binaries to this remote and store them for later use on another machine, project, or
+for sharing purposes.
+
+First, check if the remote you want to upload to is already in your current remote list:
+
+.. code-block:: bash
+
+    $ conan remote list
+
+You can search any remote in the same way you search your Conan local cache. Actually,
+many Conan commands can specify a specific remote.
+
+.. code-block:: bash
+
+    $ conan search "*" -r=my_local_server
+
+Now, upload the package recipe and all the packages to your remote. In this example, we
+are using our ``my_local_server`` remote, but you could use any other.
+
+.. code-block:: bash
+
+    $ conan upload hello -r=my_local_server
+
+This command will upload the package recipe plus all the binary packages. Using the
+``--only-recipe`` argument will upload the package recipe *only*. For fine-grained control
+over which binary packages are upload to the server, consider using the
+``--package-query/-p`` argument. ``--package-query`` allows you to explicitly declare
+which package gets uploaded to the server by specifying the package ID.
+``--package-query`` accepts a query parameter, e.g. ``arch=armv8 and os=Linux``, and only
+uploads binary packages which match this query. When using the ``--package-query`` argument, ensure
+that your query string is enclosed in quotes to make the parameter explicit to your shell.
+For example, ``conan upload <package> -p 'arch=x86_64 and os=Linux' ...`` is appropriate
+use of the ``--package-query`` argument.
```

Comment:
Maybe a bit too specific. For the tutorial, I'd probably keep this very simple, and just link the reference.